docs/descriptors/spell_cspell.md

```diff
@@ -20,20 +22,21 @@ Copy it at the root of your repository, read it, remove real spelling errors (af
-| SPELL_CSPELL_FILE_EXTENSIONS             | Allowed file extensions. `"*"` matches any extension, `""` matches empty extension. Empty list excludes all files<br/>Ex: `[".py", ""]`                                                                             | Exclude every file                              |
-| SPELL_CSPELL_FILE_NAMES_REGEX            | File name regex filters. Regular expression list for filtering files by their base names using regex full match. Empty list includes all files<br/>Ex: `["Dockerfile(-.+)?", "Jenkinsfile"]`                        | Include every file                              |
-| SPELL_CSPELL_PRE_COMMANDS                | List of bash commands to run before the linter                                                                                                                                                                      | None                                            |
-| SPELL_CSPELL_POST_COMMANDS               | List of bash commands to run after the linter                                                                                                                                                                       | None                                            |
-| SPELL_CSPELL_CONFIG_FILE                 | cspell configuration file name</br>Use `LINTER_DEFAULT` to let the linter find it                                                                                                                                   | `.cspell.json`                                  |
-| SPELL_CSPELL_RULES_PATH                  | Path where to find linter configuration file                                                                                                                                                                        | Workspace folder, then MegaLinter default rules |
-| SPELL_CSPELL_DISABLE_ERRORS              | Run linter but consider errors as warnings                                                                                                                                                                          | `false`                                         |
-| SPELL_CSPELL_DISABLE_ERRORS_IF_LESS_THAN | Maximum number of errors allowed                                                                                                                                                                                    | `0`                                             |
+| Variable | Description | Default value |
+| ----------------- | -------------- | -------------- |
+| SPELL_CSPELL_ANALYZE_FILE_NAMES | If set to `true`, MegaLinter will also send file names to cspell for analysis. Disable by defining `SPELL_CSPELL_ANALYZE_FILE_NAMES` to `false`` | `true` |
```

Comment:
@nvuillam, an extra back tick snuck in there (c.f., [the rendered doc](https://oxsecurity.github.io/megalinter/6.14.0/descriptors/spell_cspell/#configuration-in-megalinter)).

Reply:
Good catch, thanks :)